scrapers/utils.py

```diff
@@ -111,7 +112,7 @@ def validate_and_get_download_link(base_url, download_link, check_content_pdf=Tr
         response = requests.head(absolute_url)
         content_type = response.headers.get('Content-Type', '')
         is_pdf = 'application/pdf' in content_type
-        if not is_pdf:
+        if not is_pdf and download_link[-4:] != '.pdf':
             raise TypeError('Content-Type is not application/pdf.')
```

Comment:
This is probably better as well

```suggestion
            raise TypeError('File extension does not match content type.')
```

Reply:
I just committed our changed--we did add testing and were about to make one last test mocking the headers to test a PDF and a header that is plain text for our function.  I stopped when I saw your comment on this PR but committed what we had to this PR.  We can remove our testing file and just merge what we have and transfer the tests to a new branch?  Let us know how you want us to proceed

Reply:
Let's remove the testing file and do it in a separate PR, after https://github.com/sgoldfarb2/scraper-infra/pull/15 is merged